### ISSUE TITLE
Velocity plugin logging success as ERROR.

### ISSUE THREAD

**Issue by gerome1812** [GeyserMC/Floodgate#91]
On the velocity plugin, the message `INFO: Floodgate injection process succeeded!` is logged as an error.

```
[17:53:33 ERROR]: Dec 19, 2020 5:53:33 PM org.geysermc.floodgate.VelocityPlugin onProxyInitialization
[17:53:33 ERROR]: INFO: Floodgate injection process succeeded!
[17:53:33 ERROR]: Dec 19, 2020 5:53:33 PM org.geysermc.floodgate.link.SQLitePlayerLink load
[17:53:33 ERROR]: INFO: Loading Floodgate linked player database...
```

I have no other plugins than geyser and floodgate, just installed and downloaded today, dec. 19 2020.

**Comment by Camotoy** [comment]
This will be fixed in Floodgate 2.0.

**Comment by gerome1812** [comment]
Fantastic! That's the fastest response I have ever gotten on an issue.

**Comment by Tim203** [comment]
(To close it just like the other issues)
This has been fixed in Floodgate 2.0
